plugins/woocommerce-blocks/assets/js/base/components/combobox/style.scss

```diff
@@ -161,7 +158,8 @@
 
 	&.is-active,
 	&:focus-within {
```

Comment:
> When testing the Country I noticed that the is-active class is persisted after I focus outside.

@ralucaStan I am not sure why is-active is not added to state input, I spent some time testing this so I will take another look, but, the behaviour is that it retains is-active as long as the input has a value so this behaviour is expected I believe. It's important this styling is applied to everything with a value because it adjusts how the placeholder is displayed.

Reply:
@ralucaStan I think the focus class totally is used, when you focus the input the label animates to a smaller size.